docs/src/index.md

```diff
@@ -26,7 +26,7 @@ Pkg.add("https://github.com/NumericalEarth/Breeze.jl.git")
 
 A basic free convection simulation:
 
-```julia
+```@example
```

Comment:
This was changed in #105 by @glwagner, I'm still not entirely sure what's the intended use of this 😅

Reply:
In particular, see https://github.com/NumericalEarth/Breeze.jl/pull/105#discussion_r2508319005.  If running all the examples is too bothersome in all PRs we can disable them as suggested at https://github.com/NumericalEarth/Breeze.jl/issues/90#issuecomment-3506631109, and we can have some (label-based?) mechanism to run the full docs build on demand in PRs (and every time on `main` anyway)

Reply:
I believe it was changed by accident to compile the docs locally faster? @glwagner?